src/fmu/dataio/export/rms/_enums.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+from enum import Enum
+from typing import Final
+
+
+class InplaceVolumes:
+    """Enumerations relevant to inplace volumes tables."""
+
+    class Fluid(str, Enum):
+        """Fluid types"""
+
+        OIL = "OIL"
+        GAS = "GAS"
+        WATER = "WATER"
```

Comment:
I was thinking previously to change this one to have lower capital values, to match the FLUID column values. What do you think? I guess we then can use this as a type in the schema for FLUID column also 🙂

Reply:
Had to move `_enums.py` to the `export` package to accomplish this, else it tries to import `rmsapi` from within `export.rms`